code/game/gamemodes/dynamic/dynamic_rulesets.dm

```diff
@@ -111,7 +111,7 @@
 		return (threat_level >= high_population_requirement)
 	else
 		pop_per_requirement = pop_per_requirement > 0 ? pop_per_requirement : mode.pop_per_requirement
-		if(antag_cap && requirements.len != antag_cap.len)
+		if(antag_cap.len && requirements.len != antag_cap.len)
```

Comment:
won't this runtime because antag_cap is null?

Reply:
Its not null, thats the issue with the old check. It is initialized as a list
var/list/antag_cap = list()

I checked with MoMMI this time
![image](https://user-images.githubusercontent.com/33292112/67600222-ecef9480-f771-11e9-9331-477d6cf32149.png)

Reply:
You removed the default initialization above, so it will be null. no?

Reply:
I must be delirious, I dont remember doing that :eyes:

Reply:
Thanks, fixed